atlp-rwanda/atlp-devpulse-fn
# Ch Adding tailwind to the project:

## What does this PR do?

Adding and configuring tailwind. 
Css command should be given through tailwind

## Description of Task to be completed?

Using the tailwind in the project

## How should this be manually tested?

- `Clone the repo`

- `checkout` to branch `ch-add-tailwind-to-the-project`

- Pull the changes by `git pull`

- Install all packages by running `npm install`

- Start app by `running npm run dev`

- You should see the styled component using `tailwind`.

## Any background context you want to provide?

N/A

## What are the relevant pivotal tracker stories?

[Trello Board reference](https://trello.com/c/mq7DwMn6/46-add-tailwind-css-to-the-project-and-configure-it)
